fhir-profile/src/test/java/com/ibm/fhir/profile/test/BPConstraintGeneratorTest.java

```diff
@@ -21,34 +18,23 @@
 import com.ibm.fhir.registry.FHIRRegistry;
 
 public class BPConstraintGeneratorTest {
+    private static final boolean DEBUG = true;
+
     @BeforeClass
     public void before() {
         FHIRRegistry.getInstance();
         FHIRRegistry.init();
     }
 
-    public static void main(String[] args) throws Exception {
-        Logger logger = Logger.getLogger(ConstraintGenerator.class.getName());
-        logger.setLevel(Level.FINEST);
-        Handler handler = new Handler() {
-            @Override
-            public void publish(LogRecord record) {
-                System.out.println(record.getMessage());
-            }
-
-            @Override
-            public void flush() {
-                System.out.flush();
-            }
-
-            @Override
-            public void close() throws SecurityException { }
-        };
-        handler.setLevel(Level.FINEST);
-        logger.addHandler(handler);
+    @Test
+    public void generateConstraints() throws Exception {
         StructureDefinition profile = ProfileSupport.getProfile("http://hl7.org/fhir/StructureDefinition/bp");
         ConstraintGenerator generator = new ConstraintGenerator(profile);
         List<Constraint> constraints = generator.generate();
-        constraints.forEach(System.out::println);
+        for (Constraint constraint : constraints) {
+            if (DEBUG) {
+                System.out.println(constraint.id() + "\t" + constraint.expression());
```

Comment:
I think we should have some asserts here. What is actually being tested?

Reply:
sure.  the main goal with this PR is to make it run without errors.  previously it was just a main method and, when you ran it, it threw an exception.
we can't verify that the constraints compile since we don't have our FHIRPathEvaluator here.
maybe we just assert the number of constraints?

Reply:
i added the assert for the number of constraints